src/style.css

```diff
@@ -1,3 +1,73 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Ensure that the 'Roboto' font is available or included via a web font service like Google Fonts, as it may not be available on all systems by default.